articles/tokens/id-token.md

```diff
@@ -0,0 +1,88 @@
+- The signature which is used by the recipient of a JWT to validate the integrity of the information conveyed in the JWT.
+
+## How to get an ID token
+
+The `id_token` can be returned when calling any of the Auth0 functions which invoke authentication.  This includes calls to the Lock widget, to the auth0.js library, or the libraries for other languages. You can view the implementation details for retrieving the `id_token` at the [Lock web library](/libraries/lock) and [Auth0.js library](/libraries/auth0js) documents.
+
+## How to control the contents of an ID token
+
+In order to retrieve an `id_token` the `responseType` should be set to `id_token`, both for client-side and server-side authentication flows.
+
+::: panel-warning OAuth2 as a Service
+If you are not using *OAuth2 as a Service*, then you should use `code` for server side flows and `token` for client side flows.
+:::
+
+The attributes included in the issued `id_token` are controlled by the use of a [parameter called `scope`](/scopes).
+- If `scope` is set to `openid`, then the `id_token` will contain only the `iss`, `sub`, `aud`, `exp` and `iat` claims.
+- If `scope` is set to `openid name email`, then the `id_token` will contain additionally the `name` and `email` claims.
+
+If you are using Lock, the `options` object used in Lock’s instantiation can specify optional [authentication parameters](/libraries/lock/v10/customization#auth-object-) as follows:
+
+```js
+var options = {
+  auth: {
+    responseType: 'id_token',
+    params: {scope: 'openid name email'}
+  }
+};
+
+var lock = new Auth0Lock(
+  ${account.clientId},
+  ${account.namespace},
+  options
+);
+
+lock.show();
+```
+
+Again, the `responseType` must be set to `id_token` in order to get one back. The `id_token` will contain only the claims specified as the value of the `scope` parameter (in this example, `openid name email`).
+
+**NOTE:** You can also add claims to the `id_token` using [Rules](/rules), with the following format: `context.idToken['http://my-custom/claim'] = 'some-value'`.
+
+## Lifetime
+
+The `id_token` is valid for 10 hours (36000 seconds) by default.  The value can be changed in the [Dashboard > Clients > Settings](${manage_url}/#/clients/${account.clientId}/settings) screen using the `JWT Expiration (seconds)` field.
+
+There are cases where you might want to renew your `id_token`. The most common case is that you want to refresh the claims contained in the `id_token` and see that they are still valid. For example, the `id_token` may contain the user's authorization claims and you want to check that their rights are valid before allowing sensitive operations.
+
+In order to renew the `id_token` you can either perform another authorization flow with Auth0 (using the `/authorize` endpoint) or use a [Refresh Token](/tokens/refresh-token).
```

Comment:
> For example, the `id_token` may contain the user's authorization claims and you want to check that their rights are valid before allowing sensitive operations

This example might be misleading, since in many cases the ID token is not used for authorization (access token is used instead). I would remove it.